util/decompile.c

```diff
@@ -2880,8 +2880,7 @@ decompileCALLFUNCTION(int n, SWF_ACTION *actions, int maxn)
 {
 	struct SWF_ACTIONPUSHPARAM *meth, *nparam;
 
-	SanityCheck(SWF_CALLMETHOD,
-		n > 0 && actions[n-1].SWF_ACTIONRECORD.ActionCode == SWFACTION_PUSH,
+	SanityCheck(SWF_CALLMETHOD, OpCode(actions, n-1, maxn) == SWFACTION_PUSH,
```

Comment:
Where did the n> 0 condition go ?

Reply:
OpCode also performs this check, so IMO we can safely get rid of it.
```
static inline int OpCode(SWF_ACTION *actions, int n, int maxn)
{
	if(!n || n >= maxn)
	{
#if DEBUG
		SWF_warn("OpCode: want %i, max %i\n", n, maxn);
#endif
		return -999;
	} else if (n < 1) {

#if DEBUG
		SWF_warn("OpCode: want %i < 1\n", n);
#endif
		return -998;
        }
	return actions[n].SWF_ACTIONRECORD.ActionCode;
}
```